lib/pcap/parser.ex

```diff
@@ -25,7 +25,7 @@ defmodule Membrane.Element.Pcap.Parser do
         global_header: global_header,
         file: file
       }
-      ~> {:ok, &1}
+      |> then(&{:ok, &1})
```

Comment:
To be honest I don't know why the operator was there at the first place, if we don't have any pipe chain and we have just a single struct can we just put that struct directly into the `:ok`.